scripts/remove-duplicate-emails.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Script to remove users with
+ *
+ * to run:
+ * drush --script-path=../scripts/ php-script remove-duplicate-emails.php
+ */
+
+$users = db_query('SELECT * FROM users 
```

Comment:
two things
1. I would say only select UID (since we don't need the whole dang row)
2. I had trouble running [this](https://github.com/DoSomething/phoenix/blob/dev/scripts/export-users-to-northstar.php) script because that table is giant on prod, so I would suggest setting a var on the last user deleted and order by UID to start off where it died